### DISCUSSION TITLE
Unstructured Data Generation

### DISCUSSION THREAD

**Discussion by gabpmcp** [synthetichealth/synthea#1524] [General]
Hi to everyone! I have been working on a KYC project and I need a lot of unstructured synthetic data to represent DNIs from Mexico, to train a ML model for identify location data in real DNIs. What tools are effective for this task?

**Comment by jawalonoski** [comment]
KYC is an acronym for what?
DNI is an acronym for what?

**Reply by gabpmcp** [reply]
Know Your Customer and Document National Identification. I should make an OCR process on DNI and verify that user selfie seems to DNI picture.

**Reply by jawalonoski** [reply]
This project is specifically designed to generate standards-based health data. By default, the data is for the United States, but there are configurations for international locations, but Mexico is not one of them. Other than synthetic SSNs for US patients, no other national identifications are included. The synthetic patient data does not include patient photos, or actual images of any kind.
